mirdata/datasets/haydn_op20.py

```diff
@@ -0,0 +1,310 @@
+                "chords": self.chords,
+                "roman_numerals": self.roman_numerals,
+                "midi_path": self.midi_path,
+                "humdrum_annotated_path": self.humdrum_annotated_path,
+            },
+        )
+
+
+def load_score(path):
+    """Load haydn op20 score with annotations from a file with music21 format (music21.stream.Score).
+
+    Args:
+        path: path to hrm annotations
+
+    Returns:
+        music21.stream.Score: score in music21 format
+
+    """
+    if not os.path.exists(path):
+        raise IOError
+    score = music21.converter.parse(path, format="humdrum")
+    rna = list(score.flat.getElementsByClass("RomanNumeral"))
+    score.remove(rna, recurse=True)
+    return score
+
+
+def load_key(path, resolution=28):
+    """Load haydn op20 key data from a file
+
+    Args:
+        path: path to hrm annotations
+
+    Returns:
+        List[dict]: musical key data and time
+
+    """
+    if not os.path.exists(path):
+        raise IOError
+    score = music21.converter.parse(path, format="humdrum")
+    rna = {rn.offset: rn for rn in list(score.flat.getElementsByClass("RomanNumeral"))}
+    annotations = []
+    for offset, rn in rna.items():
+        if not rn:
+            continue
+        time = int(round(float(offset * resolution)))
+        tonicizedKey = rn.secondaryRomanNumeralKey
+        key = tonicizedKey or rn.key
+        annotations.append({"time": time, "key": key})
+    return annotations
+
+
+def load_midi_path(path):
+    """Load path to midi file of haydn op20 musical piece
+
+    Args:
+        path: path to hrm annotations
+
+    Returns:
+        str: midi file path
+
+    """
+    if not os.path.exists(path):
+        raise IOError
+    midi_path = os.path.splitext(path)[0] + ".midi"
+    if not os.path.exists(midi_path):
+        score = music21.converter.parse(path, format="humdrum")
+        score.write("midi", fp=midi_path)
+    return midi_path
+
+
+def load_roman_numerals(path, resolution=28):
+    """Load haydn op20 roman numerals data from a file
+
+    Args:
+        path: path to hrm annotations
+
+    Returns:
+        List[dict]: musical roman numerals data and time
+
+    """
+    if not os.path.exists(path):
+        raise IOError
+    score = music21.converter.parse(path, format="humdrum")
+    rna = {rn.offset: rn for rn in list(score.flat.getElementsByClass("RomanNumeral"))}
+    annotations = []
+    for offset, rn in rna.items():
+        if not rn:
+            continue
+        time = int(round(float(offset * resolution)))
+        figure = rn.figure
+        annotations.append({"time": time, "roman_numeral": figure})
+    return annotations
+
+
+def load_chord(path, resolution=28):
+    """Load haydn op20 chords data from a file
+
+    `Args:
+        path: path to hrm annotations
+
+    Returns:
+        List[dict`]: musical roman numerals data and time
+
+    """
+    if not os.path.exists(path):
+        raise IOError
```

Comment:
remove

Reply:
I can not.... due to the new tests.

If not this test fail:
```Python
            if dataset_name in EXCEPTIONS and method_name in EXCEPTIONS[dataset_name]:
                extra_params = EXCEPTIONS[dataset_name][method_name]
                with pytest.raises(IOError):
                    load_method("a/fake/filepath", **extra_params)
            else:
                with pytest.raises(IOError):
                    load_method("a/fake/filepath")
```

Reply:
done